### PR TITLE
Added german Names List

### PR DESCRIPTION
**Pull request by Clafter**

### CODE REVIEW

**Comment by DavideViolante** [comment]
Thank you for the contribution, could you tell me where did you get the names list? Thanks

**Comment by Clafter** [comment]
I fixed the ES-Lint Problem.

I scraped the Names from https://www.onomastik.com/Vornamen-Lexikon/

File: test/test.js [hunk 1]
```diff
@@ -50,24 +62,12 @@ describe('Tests for Gender detection from name', () => {
...
     for (const name of names) {
       assert.strictEqual(getGender(name, 'de'), 'female');
     }
   });
-  it('should return male for some names with no language', () => {
-    const names = ['Dave', 'John', 'Giacomo', 'George', 'Luca'];
-    for (const name of names) {
-      assert.strictEqual(getGender(name), 'male');
-    }
-  });
```
Comment (DavideViolante) [inline test/test.js:70]: Please restore these lines

File: test/test.js [hunk 2]
```diff
@@ -50,24 +62,12 @@ describe('Tests for Gender detection from name', () => {
...
-    for (const name of names) {
-      assert.strictEqual(getGender(name, 'de'), 'male');
-    }
-  });
   it('should return female for some names with wrong language', () => {
     const names = ['Anna', 'Jennifer', 'Lisa', 'Marta', 'Catherine'];
     for (const name of names) {
       assert.strictEqual(getGender(name, 'de'), 'female');
     }
   });
```
Comment (DavideViolante) [inline test/test.js:70]: Delete these lines

File: test/test.js [hunk 3]
```diff
@@ -50,7 +62,7 @@ describe('Tests for Gender detection from name', () => {
       assert.strictEqual(getGender(name, 'fr'), 'female');
     }
   });
-  it('should return male for some names with wrong language', () => {
+    it('should return male for some names with wrong language', () => {
```
Comment (DavideViolante) [inline test/test.js:65]:
```suggestion
  it('should return male for some names with wrong language', () => {
```

File: test/test.js [hunk 4]
```diff
@@ -62,12 +74,6 @@ describe('Tests for Gender detection from name', () => {
       assert.strictEqual(getGender(name, 'de'), 'female');
     }
   });
```
Comment (DavideViolante) [inline test/test.js:76]:
```suggestion
  });
  it('should return male for some names with no language', () => {
    const names = ['Dave', 'John', 'Giacomo', 'George', 'Luca'];
    for (const name of names) {
      assert.strictEqual(getGender(name), 'male');
    }
  });
```

**Pull request merged**